modAionImpl/src/org/aion/zero/impl/sync/Act.java

```diff
@@ -22,4 +22,8 @@
     public static final byte REQUEST_TRIE_DATA = 8;
 
     public static final byte RESPONSE_TRIE_DATA = 9;
+
+    public static final byte REQ_TX_RECEIPT_HEADERS = 10; // TODO ??
+
+    public static final byte RES_TX_RECEIPT_HEADERS = 11; // TODO ??
```

Comment:
please rename the constants to `REQUEST_RECEIPTS` and `RESPONSE_RECEIPTS` and remove the comments